lib/pluginUtils.js

```diff
@@ -50,13 +50,18 @@ function _checkDecorators (instance, decorators) {
   })
 }
 
+let anonFuncCount = 0
+
 function registerPluginName (fn) {
   const meta = getMeta(fn)
-  if (!meta) return
 
-  const name = meta.name
-  if (!name) return
-  this[registeredPlugins].push(name)
+  const functionName = meta && meta.name ? meta.name : (
+    !fn.name ? `anonymous${anonFuncCount++}` : (
+      fn.name !== 'bound _after' ? fn.name : null
```

Comment:
What is the 'bound _after' ? Could you use a constant for that?

Reply:
This is some default name for one of things `fastify` _registers_ or _uses_. Will most likely change when I re implement this feature given the new constraints